maat-court-data-api/src/main/resources/db/changelog/changes/increase_column_various_mla_tables.sql

```diff
@@ -21,4 +21,12 @@ ALTER TABLE "MLA"."XXMLA_WQ_OFFENCE" modify OFFENCE_WORDING varchar2(2500 char);
 ALTER TABLE "MLA"."XXMLA_WQ_DEFENDANT" modify ADDRESS_LINE1 varchar2(35 char);
 ALTER TABLE "MLA"."XXMLA_WQ_DEFENDANT" modify ADDRESS_LINE2 varchar2(35 char);
 ALTER TABLE "MLA"."XXMLA_OFFENCE" modify OFFENCE_WORDING varchar2(2500 char);
-ALTER TABLE "MLA"."XXMLA_DEFENDANT" modify SURNAME varchar2(50 char)
\ No newline at end of file
+ALTER TABLE "MLA"."XXMLA_DEFENDANT" modify SURNAME varchar2(50 char)
+
+-- LASB-2294 - SQL Queries executed on 8th Aug 2023
+
+ALTER TABLE "MLA"."XXMLA_RESULT" modify RESULT_TEXT varchar2(4000 char);
+
+--The following script is to roll back the changes in case of any problem :
+
+ALTER TABLE "MLA"."XXMLA_RESULT" modify RESULT_TEXT varchar2(2500 char);
```

Comment:
Perhaps it's worth putting this in a separate script for rollbacks?

Reply:
We are adding both in the same file for reference and it is easy to find if there is any issue :-)